agent/structs/config_entry.go

```diff
@@ -58,6 +58,7 @@ var AllConfigEntryKinds = []string{
 
 const (
 	BuiltinAWSLambdaExtension string = "builtin/aws/lambda"
+	BuiltinLuaExtension       string = "builtin/lua"
```

Comment:
These two constants seem duplicate to those in api/config_entry.go. Shall we remove them and change line 317 to 318 to

```
    extensions := map[string]struct{}{
        api.BuiltinAWSLambdaExtension: {},
        api.BuiltinLuaExtension:       {},
    }
```